docs/guide.md

```diff
@@ -249,14 +249,14 @@ func ProvideFoo() Foo {/* ... */}
 func ProvideBar() Bar {/* ... */}
 
 type FooBar struct {
-    Foo Foo
-    Bar Bar
+    MyFoo Foo
+    MyBar Bar
```

Comment:
Since you've renamed the fields, you will also need to update the generated code below.

Reply:
Done.